### ISSUE TITLE
Focus doesn't go to top of the screen/page for Enter your 8-digit number with VO

### ISSUE THREAD

**Issue by qanitah** [cds-snc/covid-shield-mobile#503]
With VoiceOver on, when the user goes to "Enter your 8-digit number", the focus is on "Submit" button. It should be on top of the page. Tried it multiple times and it always goes to this which is the last item on the page.

Screenshot below shows where the focus land for the VoiceOver user.

iOS 13.5.1 on iPhone 7

![image](https://user-images.githubusercontent.com/14280062/86837412-e0476e00-c06c-11ea-801f-90fb90b4250d.png)

**Comment by JuliannaR** [comment]
@smcmurtry I thought we worked on a fix for this? Thoughts?
